src/js/projects.js

```diff
@@ -0,0 +1,89 @@
+}) {
+  return `
+    <div class="project-card">
+      <img
+        class="project-card__image"
+        src="./assets/images/${imageName}"
+        alt="Snapshoot 1"
+      />
+      <div class="project-card__body">
+        <div class="project-card__header">
+          <h2 class="project-card__header-title">${title}</h2>
+          <div class="project-card__header-subtitle">
+            <span>CANOPY</span>
+            <span>Back End Dev</span>
+            <span>2015</span>
+          </div>
+        </div>
+        <p class="project-card__description">${description}</p>
+        <ul class="project-card__tags-list">
+          ${technologies.map((technology) => `<li class="project-card__tags-list-item">${technology}</li>`).join('')}
+        </ul>
+        <div class="project-card__actions">
+          <button class="button" id="project-button-${id}" type="button">See Project</button>
+        </div>
+      </div>
+    </div>
+  `;
+}
+
+const projects = [
+  {
+    title: 'Multi-Post Stories',
+    imageName: 'snapshoot-1.jpg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'js'],
+  },
+  {
+    title: 'Tonic',
+    imageName: 'snapshoot-1.jpg',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: ['html', 'css', 'js'],
+  },
+  {
+    title: 'Facebook 360',
+    imageName: 'snapshoot-1.jpg',
+    description: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    technologies: ['html', 'css', 'js'],
+  },
+  {
+    title: 'Uber Navigation',
+    imageName: 'snapshoot-1.jpg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'js'],
+  },
+  {
+    title: 'Multi-Post Stories',
+    imageName: 'snapshoot-1.jpg',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: ['html', 'css', 'js'],
+  },
+  {
+    title: 'Tonic',
+    imageName: 'snapshoot-1.jpg',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['html', 'css', 'js'],
+  },
+];
+
+portfolioSection.innerHTML = projects
+  .map((project, index) => gentateProjectHtml({ id: index, ...project }))
+  .join('');
+
+projects.forEach((project, index) => {
+  // portfolioSection.innerHTML += gentateProjectHtml({ id: index, ...project });
+
+  document.getElementById(`project-button-${index}`).addEventListener('click', () => {
+    projectModal.style.display = 'flex';
+  });
+});
+
+projectModalCloseButton.addEventListener('click', () => {
+  projectModal.style.display = 'none';
+});
```

Comment:
Hi @teckim 
As mentioned by the previous code reviewer, users still find it difficult to close the project details pop-up page. Please kindly fix this issue if not the user will find it difficult to return to the home page after clicking on the `See Project` button

Reply:
Hi @CollinsTatang, Kindly check again, because it works just find on my end.